### DISCUSSION TITLE
Manage Parameters on the Application

### DISCUSSION THREAD

**Discussion by FaitAccompli** [serverless/serverless#10322] [Q&A] [answered]
Hi, 

In the [documentation](https://www.serverless.com/framework/docs/guides/parameters), it mentions that we can manage parameters on the instances instead of the service just by going to the app section of the dashboard then selecting instance and params tab. I can't seem to find it on my dashboard? See image below. 

![image](https://user-images.githubusercontent.com/15321518/145218774-9ac9f695-9dba-4748-b744-087405d5bf86.png)

Reason is that multiple services use the same parameters and I would like to avoid redeclaring these on each service. Thank you so much for your help.

**Comment by garethmcc** [comment, marked as answer]
Parameters are deliberately scoped at the service and instance level. The reason for this is that we found having it scoped higher (app or even org) lead to teams building massive lists of parameters at the app or org level even if a parameter was only ever used at the service level which, besides being sub-optimal for security, lead to massive issues with decrypting lists of 100's of parameters at a time. Decryption is a very slow and time consuming process; the more securely something is encrypted the more complex the decryption process becomes. We also found that very few of the 100's of parameters were actually shared across all services making requiring parameters defined org or app wide not as necessary.

We made the decision to change things to the way they are now, understanding that there are some people that DO have large lists commonly used across many apps. However, there are often tough decisions to make and this was how we decided to move forward.
